docs/tutorials/kubernetes/in-cluster.md

```diff
@@ -0,0 +1,14 @@
+# In-Cluster Execution
+
+For in-cluster execution, you can use the supplied yaml to run Prowler as a job within a new Prowler namespace:
```

Comment:
What is the path of the YAML file? please use a github's permalink.

Reply:
Done